README.md

```diff
@@ -32,3 +32,8 @@ The roadmap in the [project tab](https://github.com/monstar-lab-group/admin-pane
 ## How We Work
 
 This template follows **syntax**, **semantics**, and **folder structure** defined in the [web-frontend repo](https://github.com/monstar-lab-group/web-frontend).
+
+## Logins
```

Comment:
lets add this along with the demo, as i guess it does not matter to developers as they can change the implementation and check it out always